core/templates/pages/exploration-editor-page/services/exploration-data.service.ts

```diff
@@ -149,7 +149,7 @@ export class ExplorationDataService {
           .fetchApplyDraftExplorationAsync(
             this.explorationId).then((response) => {
             this.loggerService.info('Retrieved exploration data.');
-            this.loggerService.info(JSON.stringify(response));
+            this.loggerService.info('Success');
```

Comment:
Do we need this log at all?

Reply:
@aasiffaizal in frontend tests this line is used for checking wheather data is successfully fetched or not thats why when i am removing this line then it is showing frontend test error. So if we will remove it it will not be checked that data is sussessfully fetched or not.